### PR TITLE
Updates to 64bit RTK from coreimage branch

### PR DESCRIPTION
**Pull request by micahvdm**
This should be everything. I've checked over it a few times.

### CODE REVIEW

File: setup/mod/install.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+sudo chown pistomp:pistomp /etc/authbind/byport/80
+
+#Copy WiFi hotspot files
+sudo cp hotspot/etc/default/hostapd.pistomp /etc/default
+sudo cp hotspot/etc/dnsmasq.d/wifi-hotspot.conf /etc/dnsmasq.d
+sudo cp hotspot/etc/hostapd/hostapd.conf /etc/hostapd
+sudo cp -R hotspot/usr/lib/pistomp-wifi /usr/lib
+sudo cp hotspot/usr/lib/systemd/system/wifi-hotspot.service /usr/lib/systemd/system
+sudo chown -R pistomp:pistomp /usr/lib/pistomp-wifi
+sudo chmod +x -R /usr/lib/pistomp-wif
```
Comment (rreichenbach) [inline setup/mod/install.sh:92]: typo: wif > wifi

File: setup/mod/rtkernel.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (rreichenbach) [inline setup/mod/rtkernel.sh:1]: "code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

Why is this in setup/mod?  RTK is independent of MOD.  Makes more sense to be in setup/sys.

Similar with the linux deb file.  If that file is the same as the one in michavdm 64-bit-rtk, when I run rtkernel.sh, I get the error:

+ sudo apt install ./linux-image-5.15.12-rt25-v8+_5.15.12-1_arm64.deb
Reading package lists... Done
E: Unsupported file ./linux-image-5.15.12-rt25-v8+_5.15.12-1_arm64.deb given on commandline

File: setup/mod/install.sh [hunk 3]
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```
Comment (rreichenbach) [inline setup/mod/install.sh:1]: "code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

File: setup/pedalboards/get_pedalboards.sh [hunk 4]
```diff
@@ -15,12 +15,12 @@
 # You should have received a copy of the GNU General Public License
```
Comment (rreichenbach) [inline setup/pedalboards/get_pedalboards.sh:15]: "code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

File: setup/plugins/get_plugins.sh [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (rreichenbach) [inline setup/plugins/get_plugins.sh:1]: "code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

File: setup/pkgs/gfxhat_install.sh [hunk 6]
```diff
@@ -16,9 +16,9 @@
 # along with pi-stomp.  If not, see <https://www.gnu.org/licenses/>.
 
 # GFXHat LCD
-pushd $(mktemp -d)
```
Comment (rreichenbach) [inline setup/pkgs/gfxhat_install.sh:19]: Shouldn't need to comment these out if the script is never called by setup.sh.  Will still need this for v1

**Review by rreichenbach — CHANGES_REQUESTED** [review]
Overall, looks pretty good.  Typo needs fixing.  And I couldn't get the linux deb file to load.  Need that fixed before I can fully test it.

File: setup/mod/install.sh (hunk 1 above)
Comment (micahvdm) [reply setup/mod/install.sh]: fixed

File: setup/mod/rtkernel.sh (hunk 2 above)
Comment (micahvdm) [reply setup/mod/rtkernel.sh]: I have moved the script to sys folder and re uploaded the deb file. This is the deb file I used a few days ago, so should be good now, unless GitHub upload is breaking it.

File: setup/pedalboards/get_pedalboards.sh (hunk 4 above)
Comment (micahvdm) [reply setup/pedalboards/get_pedalboards.sh:15]: the license is already there

File: setup/plugins/get_plugins.sh (hunk 5 above)
Comment (micahvdm) [reply setup/plugins/get_plugins.sh:1]: added license

File: setup/pkgs/gfxhat_install.sh (hunk 6 above)
Comment (micahvdm) [reply setup/pkgs/gfxhat_install.sh]: I have uncommented the lines in the script.

File: setup/mod/install.sh (hunk 3 above)
Comment (micahvdm) [reply setup/mod/install.sh:1]: added license

**Pull request merged**